cmd/common.go

```diff
@@ -27,6 +27,16 @@ import (
 	"github.com/spf13/viper"
 )
 
+var kubeToCISMap = map[string]string{
+	"1.11": "cis-1.3.0",
```

Comment:
I'm wondering whether we should omit the point release part of the spec number, so: cis-1.3, cis-1.4, cis-1.5 etc. Then the user doesn't need to know or care exactly when we update from, say, 1.4.1 to 1.4.2 if the CIS release an error-correcting update.

Reply:
Yes, I like this idea. Users shouldn't care about minor updates. Another benefit is less config directories.
I wonder if it makes sense to keep the kubeVersion to CIS version mapping in a `version_mapping.yaml` file.
This file could also include the **`defaultKubeVersion`** 
I think it would make it easier to update the versions down the road.